apps/wallet/src/stacks/Dashboard/TabBar.tsx

```diff
@@ -45,16 +47,18 @@ export const BottomNavigationTabBar: FC<Props> = ({ tabProps }) => {
 		[offset],
 	);
 
+	const { routes, index } = state;
+	const currentRoute = routes[index];
+
 	useEffect(() => {
-		const { routes, index } = state;
-		/* eslint-disable-next-line */
-		const currentRoute: any = routes[index];
-		const widgetId = currentRoute.params?.params?.id;
-		const isExploreTab = currentRoute.name === 'Explore' && !widgetId;
+		const mustHaveBottomTabBar = mustHaveBottomTabBarRoutes.includes(
+			currentRoute.name,
+		);
+
+		const nextOffset = isDrawerOpen || mustHaveBottomTabBar ? 0 : realBarHeight;
```

Comment:
would this cover `Widget Explorer` screen? this doesn't seem related or part of this Ticket anyway.

Reply:
> would this cover `Widget Explorer` screen? this doesn't seem related or part of this Ticket anyway, not even related.

because it has an impact on the way i change the navigation of collection and collectible inside a widget so that i have to adjust it
the `Widget Explorer` is not included in the `TabBar` so I think we don't need to cover it